user_docs/en/changes.t2t

```diff
@@ -56,6 +56,7 @@ Add-ons will need to be re-tested and have their manifest updated.
 - The ``BrailleDisplayDriver`` base class now has ``numRows`` and ``numCols`` properties to provide information about multi line braille displays.
 Setting ``numCells`` is still supported for single line braille displays and ``numCells`` will return the total number of cells for multi line braille displays. (#15386)
 - Updated BrlAPI for BRLTTY to version 0.8.5, and its corresponding python module to a Python 3.11 compatible build. (#15652, @LeonarddeR)
+- The file names of the NVDA Controller Library no longer contain a suffix denoting the architecture, i.e. ``nvdaControllerClient32/64.dll`` are now called ``nvdaControllerClient.dll``. (#15718, #15717, @LeonarddeR)

```

Comment:
should this be under API breaking changes?

Reply:
I don't think so. This doesn't break a particular API, i.e. the old controller client can still be used and nothing is changed about the internals of the controller, so people don't need updating. The only thing that changes is the distribution.

Reply:
Actually I'm planning to provide a follow up that will be API breaking anyway, so we can revisit the changelog as part of that pr. Is that ok with you? Code is mostly written but blocked by #15699